src/Controls/tests/UITests/Tests/Issues/Issue18753.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue18753 : _IssuesUITest
+	{
+		public Issue18753(TestDevice device) : base(device) { }
+
+		public override string Issue => "[D8] Editor IsTextPredictionEnabled works";
+
+		[Test]
+		public void Issue18753Test()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Mac, TestDevice.Windows },
+				"Currently IsKeyboardShown is not implemented.");
+
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android },
```

Comment:
On Android should be working by this recent fix https://github.com/dotnet/maui/issues/8558
But, are getting suggestions with a IsTextPredictionEnabled to false.
![image](https://github.com/dotnet/maui/assets/6755973/48ed3f25-5425-4dba-8778-ecc6ce0f5d2a)